bindata/oauth-apiserver/deploy.yaml

```diff
@@ -101,18 +101,36 @@ spec:
         - mountPath: /var/log/oauth-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 10
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 3
         readinessProbe:
-          failureThreshold: 10
           httpGet:
             scheme: HTTPS
             port: 8443
             path: readyz
-      terminationGracePeriodSeconds: 70 # a bit more than the 60 seconds timeout of non-long-running requests
+          initialDelaySeconds: 0
+          periodSeconds: 5
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 1
+        startupProbe:
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 5
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 30
```

Comment:
same questions as above: how did we come up with the custom thresholds for `periodSeconds` and `failureThreshold`?

Reply:
so, effectively this means: "tolerate 30 failures during startup, and up to 3 failures once started before killing"

Reply:
> so, effectively this means: "tolerate 30 failures during startup

Yes, this allows a long startup.  Chosen based on a comment somewhere from sttts saying it took 10 seconds per API and 100 seconds total.

> up to 3 failures once started before killing"

Once started, it uses liveness checks

> same questions as above: how did we come up with the custom thresholds for periodSeconds and failureThreshold?

it roughly matches the long timeouts from before that were slowing down responsiveness overall.